server/src/main/java/org/candlepin/model/Pool.java

```diff
@@ -117,8 +117,12 @@
         /** Attribute used to determine which specific host the pool was created for */
         public static final String REQUIRES_HOST = "requires_host";
 
-        /** Attribute used to determine if a pool was created by sharing */
-        public static final String SHARE = "share_derived";
+        /** Attribute used to determine if a pool was created by a share */
+        public static final String SHARED_POOL = "shared_pool";
```

Comment:
Is this comment accurate? You have a CREATED_BY_SHARE. How is SHARED_POOL different?.